src/PaddedViews.jl

```diff
@@ -186,6 +202,66 @@ outerinds() = error("must supply at least one array with concrete axes")
 padrange(i1::OneTo, i2::OneTo) = OneTo(max(last(i1), last(i2)))
 padrange(i1::AbstractUnitRange, i2::AbstractUnitRange) = min(first(i1),first(i2)):max(last(i1), last(i2))
 
+
+"""
+    Aspad = cpaddedviews(fillvalue, A1, A2, ....)
```

Comment:
I wonder if the `c` will be opaque. What about calling this `sym_paddedviews` or `symmetric_paddedviews` or `centered_paddedviews` or `centeredviews`?

Reply:
Ha, the names. I was thinking about dispatch `paddedviews` on trait but failed to find a good name candidate for the previous behavior.

`sym_paddedviews` looks good to me, will change to that.

I came up with `centered_paddedviews` first but find it quite lengthy; (take the example of my preview script, it can't be fit in one line)

`centeredviews` seems not the right name; confuses with `ImageFiltering.centered`